src/js/typescript.js

```diff
@@ -81,9 +81,11 @@ export default class TypeScriptCompiler extends SimpleCompilerBase {
   addHotModuleLoadingRegistration(sourceCode, fileName, exports) {
     if (exports.length < 1) return sourceCode;
 
-    let registrations = exports.map(x => 
-      `__REACT_HOT_LOADER__.register(${x}, "${x}", __FILENAME__);\n`
-    );
+    let registrations = exports.map(x => {
+      let id = `${x}` == 'default' ? "_default" : `${x}`
```

Comment:
So, I upgraded to the latest electron-compilers (away from my fork) and.. for some reason, with typescript & es2017 target, this breaks. I'm not really sure what happened, maybe some caching foiled my earlier testing?

Anyhoo it works replacing "_default" with "exports.default", but that depends on the typescript module target I suppose. I'd make a PR but who knows if that'd break it for someone else?

Note that I'm using passthrough for "text/javascript" (without that, electron-compile was trying to process my entire `node_modules` it seems!)